modules/default/calendar/calendar.js

```diff
@@ -122,6 +122,9 @@ Module.register("calendar", {
 
 	// Override socket notification handler.
 	socketNotificationReceived: function (notification, payload) {
+		if (this.identifier != payload.id)
```

Comment:
should be !== to check types too

Reply:
fixed

Reply:
Thx. Also you should fix the tests, please run "npm run test:js" to see what can be fixed

Reply:
Did it. 

The contribution guidelines mention `grunt`, but there is no `gruntfile` on the development branch. Am i supposed to do it by hand or is there any other tool to use?

Reply:
Grunt is removed on the develop branch so maybe we should remove it from the guidelines. Where is it mentioned exactly?

Reply:
Here: https://github.com/MichMich/MagicMirror/blob/develop/.github/CONTRIBUTING.md

If you open a pull request for teh first time github points you there.

Reply:
Ah, right. That points to the master branch, which still mentions grunt. Once develop is merged with release 2.12 that will be fixed... Thanks for the PR and the tip.